Cubical/Categories/Category/Base.agda

```diff
@@ -51,40 +52,63 @@ comp' = _∘_
 infixr 16 comp'
 syntax comp' C g f = g ∘⟨ C ⟩ f
 
+
 -- Isomorphisms and paths in categories
-record CatIso (C : Category ℓ ℓ') (x y : C .ob) : Type ℓ' where
-  constructor catiso
+
+record isIso (C : Category ℓ ℓ'){x y : C .ob}(f : C [ x , y ]) : Type ℓ' where
```

Comment:
Maybe we should have no-eta-equality for this? Might be good to avoid slow typechecking times

Reply:
Oh, the extra judgemental equality is very useful, e.g. [symmetry of inverse](https://github.com/agda/cubical/blob/cc8812edc0d3ac57e60cfebe5dd42c49fcb773ff/Cubical/Categories/Morphism.agda#L129), and I think it is used in many places in the library.

Reply:
The typechecking time is not a problem so far, do you think it is necessary to change it?

Reply:
Ah ok, if it's used several times and simplifies proofs but doesn't lead to any problems for typechecking times then we should not remove it